extensions/resource-deployment/src/services/tools/toolBase.ts

```diff
@@ -26,6 +26,22 @@ const toolStatusLocalized: Map<ToolStatus, string> = new Map<ToolStatus, string>
 	[ToolStatus.Failed, toolStatusFailed]
 ]);
 
+export const enum InformationalMessageType {
```

Comment:
I'd prefer to make it specific indicating dependencies instead of general informationalMessageType

Reply:
changed.